src/chainbridgeConfig.ts

```diff
@@ -75,7 +75,7 @@ export const chainbridgeConfig: ChainbridgeConfig = {
       ],
     },
     {
-      chainId: 1,
+      chainId: +((process.env.REACT_APP_CERE_CHAIN_ID as unknown) as number),
```

Comment:
Why did you change it?

Reply:
Because we need to pass different value for Testnet and Mainnet